src/components/Spinner.js

```diff
@@ -8,13 +9,15 @@ import { colors, position } from '@rainbow-me/styles';
 
 const Spinner = ({ color, duration, size, ...props }) => (
   <Centered {...props}>
-    <SpinAnimation duration={duration}>
-      <ImgixImage
-        source={SpinnerImageSource}
-        style={position.sizeAsObject(size)}
-        tintColor={color}
-      />
-    </SpinAnimation>
+    {!IS_TESTING && (
```

Comment:
Remember that unfortunately this is a string so it should be 
`IS_TESTING !== 'true'`

Reply:
true!! i had it right and then i moved into the actual component. will fix pre-merge